conda-incubator/conda-store
# Add user journey test

Fixes #668 
Fixes #662




## Description
Adds an initial user journey test. This test follows a normal user workflow (using only api calls) to ensure that user workflows are not broken. It also begins to build up a library of actions that can be used to compose future user journey tests.

This test has been tested locally, in CI and against https://nebari.quansight.dev and it worked in all cases.

## Pull request checklist




## Additional information


## How to test
1. start conda store locally `docker compose up`
2. in another tab navigate to the `conda-store-server` folder `cd conda-store-server`
3. run the test `pytest -m user_journey`
4. test should pass
